eap-xp4-jdk17/tests/features/sso.feature

```diff
@@ -0,0 +1,32 @@
+@jboss-eap-7
+Feature: EAP XP SSO tests
```

Comment:
Just curious if you're running these tests? Aren't we now explicitly excluding SSO?

Reply:
@xstefank the test runs when the image is added to an errata. I think we need to addapt the test to no SSO